tenantResolver.js

```diff
@@ -4,12 +4,20 @@ var passport = require('passport');
 var OidcStrategy = require('passport-openidconnect').Strategy;
 var logger = require('./logger')
 
-const defaultTenantSub = "default"
+const defaultTenantSub = "default";
+
+var defaultTenantConfig = {
```

Comment:
This is handled automatically by providing a null config, see Tenant constructor

Reply:
The env you have for OKTA_ORG_URL should be TENANT, i think the rest should then just work. I'll commit an example env file. Sorry this should have been in the inital commit.

Reply:
Correct, thanks, removed the code